src/cargo/core/compiler/context/compilation_files.rs

```diff
@@ -49,6 +49,16 @@ pub struct OutputFile {
     pub flavor: FileFlavor,
 }
 
+impl OutputFile {
+    /// Gets the hardlink if present. Otherwise returns the path.
+    pub fn bindst(&self) -> &PathBuf {
```

Comment:
Perhaps a `_` in the middle to separate "bin" and "dst"?

Reply:
I'll follow-up PR.